src/sync/gll.ts

```diff
@@ -0,0 +1,42 @@
+import PQueue from 'queue-promise';
+import { ILogger } from '../types';
+import { GllTaskError } from '../errors';
+
+let GLL = new PQueue({ concurrent: 5 });
```

Comment:
Move `5` to a constant, maybe `MAX_CONCURRENT_LOAD_TASKS`?

Reply:
Done

Reply:
Thanks!